index.js

```diff
@@ -45,7 +45,9 @@ class StackUtils {
     return [...natives];
   }
 
-  clean (stack) {
+  clean (stack, indent = 0) {
```

Comment:
use of a default argument broke node 4, so this was a breaking change. see https://github.com/tapjs/stack-utils/commit/e0d38e771d49bb6f4ca3752267b0e5774cef5829#r48906765